charts/jenkins/templates/jenkins-backup-cronjob.yaml

```
@@ -29,6 +29,15 @@ spec:
         spec:
           restartPolicy: OnFailure
           serviceAccountName: {{ template "jenkins.fullname" . }}-backup
+{{- if .Values.backup.usePodSecurityContext }}
+          securityContext:
```

To give an example what I mean:

```suggestion
          {{- if .Values.backup.usePodSecurityContext }}
          securityContext:
```

---

sure @torstenwalter, thanks for the suggestions